tests/components/tplink/test_light.py

```diff
@@ -287,7 +287,7 @@ async def test_smartswitch(
 
     state = hass.states.get("light.dimmer1")
     assert state.state == "on"
-    assert state.attributes["brightness"] == 48.45
+    assert state.attributes["brightness"] == 51
```

Comment:
Shouldn't this be 49?

Reply:
50 round tripping though has a loss of precision of 1 going though the conversion where is previously had a loss of 2